src/V3Width.cpp

```diff
@@ -7826,6 +7826,17 @@ class WidthVisitor final : public VNVisitor {
             if (patp->keyp()) {
                 if (const AstConst* const constp = VN_CAST(patp->keyp(), Const)) {
                     element = constp->toSInt();
+                } else if (const AstText* textp = VN_CAST(patp->keyp(), Text)) {
+                    // Check for decoded Text nodep                
+                    if(patp->nodep()){
+                        const AstVar* varp = VN_AS(patp->nodep(), Var);
+                        UINFO(9, " textp " << textp <<  endl);
+                        UINFO(9, " varp " << varp <<  endl);
+
+                        const AstConst* constp = VN_AS(varp->valuep(), Const);
+                        element = constp->toSInt();
+                        UINFO(9, " constp " << constp <<  endl);
+                    }
```

Comment:
I think now we have a VarRef it will become a Const and this shouldn't be needed - check the .tree files to make sure.

Reply:
I'm not seeing an automatic conversion of the VarRef to Const, at the stage when W3Width wants to use it. Calling `V3Const::constifyParamsEdit` directly does.

Reply:
Ok, that is reasonable to call.